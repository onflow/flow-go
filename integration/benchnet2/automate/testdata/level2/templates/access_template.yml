{{range $val := .}}args:
  - "--bootstrapdir=/bootstrap"
  - "--access"
env:
  - name: access name
    value: access {{$val.NodeID}}
  - name: access numbers
    value: 123{{- end -}}
