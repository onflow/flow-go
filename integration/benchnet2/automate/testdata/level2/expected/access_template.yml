args:
  - "--bootstrapdir=/bootstrap"
  - "--access"
env:
  - name: access name
    value: access 12345
  - name: access numbers
    value: 123
